src/test_workflow/integ_test/distribution_tar.py

```diff
@@ -25,6 +25,10 @@ def install_dir(self) -> str:
     def config_path(self) -> str:
         return os.path.join(self.install_dir, "config", self.config_filename)
 
+    @property
+    def data_dir(self) -> str:
+        return os.path.join(self.install_dir, "data")
+
```

Comment:
@peterzhuamazon  Is the data directory not removed for `tar` and `zip`?

Reply:
No need for tar and zip due to their data folder is within the install_dir, unlike deb/rpm which existed in the /var/lib/<> location.